src/frontend/Komponenter/Behandling/VedtakOgBeregning/Barnetilsyn/OppsummeringAvBarn.tsx

```diff
@@ -80,63 +88,79 @@ export const OppsummeringAvBarn: React.FC<{
-                                    {formaterIsoDato(barnepassordning.til)}
-                                </BodyShortSmall>
-                            </GridLinje>
-                            <NedersteGridLinje>
-                                <IkonOgTekstWrapper>
-                                    <Søknadsgrunnlag />
-                                    <MarginTekst>Utgifter per måned</MarginTekst>
-                                </IkonOgTekstWrapper>
-                                <BodyShortSmall>{barnepassordning.beløp},-</BodyShortSmall>
-                            </NedersteGridLinje>
+                            {søkesOmBarnetilsynForBarn ? (
+                                <>
+                                    <GridLinje>
+                                        <IkonOgTekstWrapper>
+                                            <Søknadsgrunnlag />
+                                            <BoldTekst size="small">Barnepassordning</BoldTekst>
+                                        </IkonOgTekstWrapper>
+                                        <Label size="small">
+                                            {typeBarnepassordningTilTekst[barnepassordning.type]}
+                                        </Label>
+                                    </GridLinje>
+                                    <GridLinje>
+                                        <IkonOgTekstWrapper>
+                                            <Søknadsgrunnlag />
+                                            <MarginTekst>Navn passordning</MarginTekst>
+                                        </IkonOgTekstWrapper>
+                                        <BodyShortSmall>{barnepassordning.navn}</BodyShortSmall>
+                                    </GridLinje>
+                                    <GridLinje>
+                                        <IkonOgTekstWrapper>
+                                            <Søknadsgrunnlag />
+                                            <MarginTekst>Periode passordning</MarginTekst>
+                                        </IkonOgTekstWrapper>
+                                        <BodyShortSmall>
+                                            {formaterIsoDato(barnepassordning.fra)} -{' '}
+                                            {formaterIsoDato(barnepassordning.til)}
+                                        </BodyShortSmall>
+                                    </GridLinje>
+                                    <NedersteGridLinje>
+                                        <IkonOgTekstWrapper>
+                                            <Søknadsgrunnlag />
+                                            <MarginTekst>Utgifter per måned</MarginTekst>
+                                        </IkonOgTekstWrapper>
+                                        <BodyShortSmall>{barnepassordning.beløp},-</BodyShortSmall>
+                                    </NedersteGridLinje>
+                                </>
```

Comment:
Kunne hatt en grid for hele slik at ikke hver rad blir selvstendig (samme som andre steder). Mister dog litt av oversiktligheten man får i koden ved at man ser når neste rad begynner

Reply:
Dette er nå utbedret. I tillegg til at FlexBox og Grid tar seg av spacing mellom komponenter: [f056942](https://github.com/navikt/familie-ef-sak-frontend/pull/2141/commits/f0569424230d9abcc4768bf630abe130687b7352)